src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -8436,18 +8443,6 @@ void MusicXmlParserNotations::technical()
 
 void MusicXmlParserNotations::otherTechnical()
 {
-    const Color color = Color::fromString(m_e.attribute("color"));
```

Comment:
color is still needed a few lines furter down

Reply:
Happened during rebase.